PLEP-0001.md

```diff
@@ -0,0 +1,176 @@
+`1.0.0`.  PLEPs are not needed for minor changes.  
+
+All new PLEPs should begin with a copy of the PLEP template contained
+within the PlasmaPy/PlasmaPy-PLEPs repository.  This copy should be
+renamed to `PLEP-nnnn.md` where `nnnn` is the lowest available PLEP
+number preceded by zeros.  The template contains a suggested outline
+that is most appropriate for a standard PLEP.  Not all sections
+included in the template are required for every PLEP, and sometimes it
+is appropriate to create different sections.  PLEPs are written in
+Markdown.
+
+### Amending or superceding a PLEP
+
+PLEPs may be amended or superceded as PlasmaPy grows and changes.
+
+If a topic is already covered by an existing PLEP, then it is often
+appropriate to propose an amendment to that PLEP instead of writing a
+new PLEP.  Amendments should not substantially change the spirit of
+the PLEP.  Proposed amendments should update the "date last revised"
+category in the header.  Amendments to PLEPs go through the same
+process as newly proposed PLEPs.
+
+When major changes to a PLEP are desired, then it is most appropriate
+to propose a new PLEP to supercede the old PLEP.  The pull request to
+supersede a PLEP should change the "status" category in the header of
+the old PLEP to "superseded by PLEP nn" where nn is the number of the
+new PLEP.
+
+When PLEPs are amended or superseded, it is generally helpful to
+request reviews from the authors of the original PLEP.
+
+Minor changes to a PLEP that do not affect its meaning or intent
+(e.g., formatting changes, fixes to typos, updates to links, minor
+rewording to improve clarity, and reversing cosmic ray bit flips) do
+not need to go through the full review process, but do require
+approval by the Coordinating Committee.
+
+### Submitting a PLEP
+
+All new PLEPs and amendments to PLEPs should be submitted as pull
+requests into the PlasmaPy-PLEP repository.  The pull request may be
+created while the PLEP is being written to allow for greater
+transparency and community input during the writing process.  In this
+case, the sponsor should comment on the pull request and inform the
+Coordinating Committee when the proposed PLEP is ready for review.
+
+### Review process
+
+When a new PLEP or an amendment to a PLEP is submitted, the
+Coordinating Committee should appoint one of its members to be the
+PLEP's editor.  The editor is responsible for aiding the sponsor by
+making sure that the PLEP follows the accepted standard and
+facilitating communication between the sponsor and the Coordinating
+Committee.  The editor and sponsor may be the same person.
+
+When the editor agrees that the PLEP is reasonably close to the
+accepted standard, then the sponsor, the editor, or another member of
+the Coordinating Committee should email the PlasmaPy list to introduce
+the PLEP and request community feedback.  The discussion should take
+place through normal communication channels.  Important points and
```

Comment:
I appreciate these may change, but is it worth saying here what the 'normal communication channels' are? Or perhaps linking to some kind of 'normal contact channels' page on the website, which we would keep up to date.